imagetest/utils/test_utils.go

```diff
@@ -88,3 +97,15 @@ func DownloadGCSObjectToFile(ctx context.Context, client *storage.Client, gcsPat
 	}
 	return nil
 }
+
+// IsTargetLinux check if the vm is target linux distribution.
+func IsTargetLinux(version string) bool {
+	imageMetadata, err := GetMetadata("image")
+	if err != nil {
+		log.Fatalf("Failed to get image from metadata server: %v\n", err)
```

Comment:
don't use log package in tests, and you should be returning an error so caller can determine how to handle.
and since they're handling that anyway, i think they can just call getmetadata directly, what do you think?

Reply:
Agree to remove this util function and directly write it in test.